packages/headless/src/features/instant-results/instant-results-slice.ts

```diff
@@ -74,6 +84,8 @@ const makeEmptyCache = (
     error: null,
     results: [],
     expiresAt: 0,
+    isActive: false,
```

Comment:
This flag is meant to represent when there is a focus on the search box VS blur ?